src/main/java/org/spongepowered/mod/text/SpongeTextFactory.java

```diff
@@ -54,6 +55,10 @@ public Text parseJson(String json) throws IllegalArgumentException {
         }
     }
 
+    public static Locale getDefaultLocale() { // TODO: Get this from the MC client?
+        return Locale.getDefault();
```

Comment:
It would be cool to have a config for this.
Especially for servers which are hosted on big server providers without the ability to change the systems locale.